src/lib/deploy.js

```diff
@@ -42,6 +42,16 @@ export async function deploy({ alias, yes }) {
   const latestCliVersion = await getLatestCliVersion();
   const installedCliVersion = getInstalledCliVersion();
 
+  if (!installedCliVersion) {
+    log(
+      chalk.red(
+        `Failed to detect the installed zkapp-cli version. This might be possible if you are using Volta to manage your Node versions.`
+      )
+    );
+    log(chalk.red('As a workaround, you can as zkapp-cli as a local dependency by running `npm install zkapp-cli`'));
```

Comment:
@satyambnsal thank you for PR!

> you can as zkapp-cli as a local dependency

You meant "you can install ..."?

Also I doubt "\`npm install zkapp-cli\`" will be rendered as the code snippet, so you can remove that part completely or highlight it with something like `chalk.bold` perhaps or similar.

Reply:
@shimkiv thank you for the feedback. making the changes.

Reply:
Changes done. Also I see that other commands in file are also using **``** syntax, so I've kept it same.